<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChemFlow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_logo_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ChemFlow</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>591</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_experiment">
     <attribute name="title">
      <string>Experiment</string>
     </attribute>
     <widget class="QLabel" name="label_protocol">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Protocol</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_lig">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dockingSoftware">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AutoDock Vina</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PLANTS</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBox_lig">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_inputFiles">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Input files</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rescoring">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>359</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rescoring</string>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>309</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_forceFieldMin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>309</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Force field minimisation</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_minConfigure">
      <property name="geometry">
       <rect>
        <x>241</x>
        <y>329</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_readConfig">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>458</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Read config file</string>
      </property>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>359</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>329</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_runConfigure">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>86</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_output">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_writeConfig">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>458</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Write config file</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_lig">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ligand(s)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_12">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>10</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_min">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Minimize the complex </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dockingKeep_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>269</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>pose(s) per ligand</string>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>239</y>
        <width>20</width>
        <height>71</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_protocolPreset">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Docking</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Docking + Rescoring</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rescoring</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_outputDir">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Output directory</string>
      </property>
     </widget>
     <widget class="Line" name="line_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>379</y>
        <width>20</width>
        <height>68</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_protocolPreset">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>31</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Preset</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_rec">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_rescoringKeep_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>408</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>pose(s) per ligand</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_runOptions">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Locally</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PBS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Slurm</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_dockingConfigure">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>86</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>431</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_lig">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dockingSoftware">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>241</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Software</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_rescoringSoftware">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>379</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AutoDock Vina</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MM/PBSA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PLANTS</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_dockingKeep">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>268</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Keep</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_run">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>458</y>
        <width>86</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rescoringSoftware">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Software</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_rescoring">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>408</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>458</y>
        <width>86</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_docking">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Docking</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>20</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>438</y>
        <width>551</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_docking">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>269</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="Line" name="line_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>20</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_output">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>190</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_rec">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Receptor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_runOptions">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>61</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rec">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_output">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rescoringConfigure">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>379</y>
        <width>86</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rescoringKeep">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>407</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Keep</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_report">
     <attribute name="title">
      <string>Report</string>
     </attribute>
     <widget class="QLabel" name="label_analyse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Analyse</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_configureExp">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>334</x>
        <y>10</y>
        <width>86</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_analyse">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Current experiment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Another experiment</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_reportPreset">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>41</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Preset</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_reportPreset">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>330</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Docking</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Docking benchmark: RMSD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Docking + Rescoring benchmark: actives and decoys</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
